dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/globalid/DOIDataCiteRegisterServiceTest.java

```diff
@@ -148,6 +149,7 @@ void getMetadataForDeactivateIdentifier() {
                                                                       "        <description descriptionType=\"Abstract\">:unav</description>\n" +
                                                                       "    </descriptions>\n" +
                                                                       "    <contributors></contributors>\n" +
+                                                                      "    \n" +
```

Comment:
Mayne we could add some unit test that will check if we correctly include fundingreference and ror element into creators